hacspec/src/lib.rs

```diff
@@ -60,9 +60,9 @@ pub fn r_process_message_1(
         let (method, supported_suites, g_x, c_i) = parse_message_1(message_1);
 
         // verify that the method is supported
-        if method.declassify() == EDHOC_METHOD {
+        if method == EDHOC_METHOD {
             // Step 2: verify that the selected cipher suite is supported
-            if supported_suites[0u8].declassify() == EDHOC_SUPPORTED_SUITES[0u8].declassify() {
+            if supported_suites[0] == EDHOC_SUPPORTED_SUITES[0u8].declassify() {
```

Comment:
Let's homogenize this so `declassify` does not appear any more in `*_process_message`?